<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RomalLoptop" ignored="1" name="Default test(failed)(failed)" tests="1" failures="1" timestamp="2020-09-21T01:02:44 EDT" time="9.079" errors="0">
  <testcase name="validLogin" time="1.973" classname="com.hrms.testcases.LoginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.hrms.testcases.LoginTest.validLogin(LoginTest.java:16)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validLogin -->
  <testcase name="invalidLogin" time="0.0" classname="com.hrms.testcases.LoginTest">
    <ignored/>
  </testcase> <!-- invalidLogin -->
</testsuite> <!-- Default test(failed)(failed) -->
